src/phoenix/trace/fixtures.py

```diff
@@ -146,6 +214,50 @@ def load_example_traces(use_case: str) -> TraceDataset:
     return TraceDataset(json_lines_to_df(download_traces_fixture(fixture)))
 
 
+def get_dataset_fixtures(
+    use_case: str,
```

Comment:
What's the difference between `use_case` and `fixture_name`?

Reply:
I think this is legacy, I just copied-pasted this.